_locales/uk/messages.json

```diff
@@ -0,0 +1,748 @@
+{
+	// DKIM_STRINGS
+	"loading": {
+		"message": "Перевірка..."
```

Comment:
Should be the special `…` ellipsis character instead of the three dots. Same in the other places.